library/lib/polyfill.js

```diff
@@ -0,0 +1,5 @@
+export default function polyfill(features = [], { minify = true } = {}) {
+  return minify
```

Comment:
Why would you need this boolean switch?

Reply:
Whenever you need to debug a polyfill (been there) it's easier to debug unminified code.

Reply:
Shouldn't we make it min when NODE_ENV=production? Just like we do with our own code?

Reply:
Well that makes perfectly sense.